querydsl-mongodb/src/main/java/com/querydsl/mongodb/MongodbSerializer.java

```diff
@@ -256,6 +256,9 @@ public Object visit(Operation<?> expr, Void context) {
         } else if (op == MongodbOps.NEAR) {
             return asDBObject(asDBKey(expr, 0), asDBObject("$near", asDBValue(expr, 1)));
 
+        } else if (op == MongodbOps.NEARSPHERE) {
+            return asDBObject(asDBKey(expr, 0), asDBObject("$nearSphere", asDBValue(expr, 1)));
```

Comment:
If `$elemMatch` becomes `ELEM_MATCH`, isn't it more consistent to use `NEAR_SPHERE` for `$nearSphere`?